daemon/volumes.go

```diff
@@ -52,6 +52,9 @@ func (m *mountPoint) Setup() (string, error) {
 			if !os.IsNotExist(err) {
 				return "", err
 			}
+			if _, err := os.Stat(m.Source); err != nil && os.IsNotExist(err) {
```

Comment:
I suppose we can, though this would picked up by the subsequent `os.MkdirAll` anyway.
But since we're already adding the stat call, might as well handle earlier.